mne_bids/tests/test_mne_bids.py

```diff
@@ -1,22 +1,24 @@
-""" Testing MNE BIDS converter
+"""Test the MNE BIDS converter.
 
-There are four supported MEG systems in MNE.
-We should test their functionality with their sample data.
+For each supported file format, implement a test.
 """
 # Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
 #          Teon L Brooks <teon.brooks@gmail.com>
 #          Chris Holdgraf <choldgraf@berkeley.edu>
+#          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#
 # License: BSD (3-clause)
 
 import os
 import os.path as op
+from warnings import simplefilter
 
 import mne
 from mne.datasets import testing
 from mne.utils import _TempDir, run_subprocess
 
-from mne_bids import raw_to_bids
-from warnings import simplefilter
+from mne_bids.mne_bids import raw_to_bids
```

Comment:
any reason to not just use `from mne_bids import raw_to_bids` here?

Reply:
no reason, let's change it to what you suggest